Documentation/Introduction/Index.rst

```diff
@@ -17,7 +17,7 @@ publication):
 
 .. image:: ../Images/overview.png
    :alt: Protecting a folder and a few individual files
-   :align: center
+   :class: with-border with-shadow
```

Comment:
center is important if you generate a PDF, for `with-border with-shadow`, please show how it will look like by providing a URL of a documentation using it, to understand what it does.

Reply:
* Where to find the link to the DemoDocs ("theme stress test"): https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/

* https://docs.typo3.org/m/typo3/demo-t3SphinxThemeRtd/master/en-us/ImagesAndFigures/Index.html#bright-images-with-border-and-shadow 

* Note that `:align: center` does not affect html.

* Interesting in this context: Directive 'centered' is deprecated. The advice is to use directive 'rst-class' instead.

* I'll soon be adding css classes `centered` and `align-center`: 
![image](https://user-images.githubusercontent.com/307057/98533583-7261bc00-2283-11eb-9572-8ceee6080762.png)